source/parser.hera

```diff
@@ -4025,8 +4025,16 @@ IfStatement
     }
 
 ElseClause
-  Nested Else BlockOrEmpty
-  _? Else BlockOrEmpty
+  Nested Else BlockOrEmpty:block -> {
```

Comment:
The could be combined into `( Nested / _? ) Else BlockOrEmpty:block` to prevent a little backtracking.